app/app.go

```diff
@@ -652,8 +689,24 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 	paramsKeeper.Subspace(ibctransfertypes.ModuleName)
 	paramsKeeper.Subspace(ibchost.ModuleName)
 	paramsKeeper.Subspace(evmtypes.ModuleName)
+	paramsKeeper.Subspace(gravitytypes.ModuleName)
 	// this line is used by starport scaffolding # stargate/app/paramSubspace
 	paramsKeeper.Subspace(cronosmoduletypes.ModuleName)
 
 	return paramsKeeper
 }
+
+// VerifyAddressFormat verifis the address is compatible with ethereum
```

Comment:
`verifies the address is compatible with ethereum`

For now it just check the length, is it enough?

Reply:
I guess so, any bytes with length of 20 is a valid ethereum address right?

Reply:
yes, should be fine. 

Go ethereum would accept any length and just do cropping but at the end length will be 20

Reply:
```suggestion
// VerifyAddressFormat verifies the address is compatible with ethereum
```